### ISSUE TITLE
Error toasts need to be dismissable

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4058]
## Area or Component
Error notifications across the app (usually on form submission, like facility form or patient form)

## Quick Summary
The existing toast pattern is not accessible - users need to be able to dismiss the error notification, rather than having it drop automatically.

## Accessibility Violation
For error identification, it's best practice to tell users a summary of the issue before moving them to the first error: https://www.w3.org/TR/WCAG21/#error-identification

## Action Requested
Make the error toast dismissable

**Comment by kenieh** [comment]
### Design

<img width="376" alt="Screen Shot 2022-07-25 at 11 06 43 AM" src="https://user-images.githubusercontent.com/72819285/180844718-ad5b4446-ea7c-44b9-8504-93c5756024c6.png">

**FIGMA:**
https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=12545%3A34043

**Comment by emyl3** [comment]
@kenieh Currently, we have set a behavior on these toast notifications to auto close after 5 seconds. With the addition of a close button, do you want us to keep the auto close behavior or remove it entirely?

**Comment by emyl3** [comment]
Decisions made w/ Kenny on 09/30/2022:
- Keep auto-close behavior at 5 seconds
- If possible move the toast message to appear on the bottom left corner (if effort to do so is low)
- Okay to use the uswds icon for the "Error" status  (a circle information icon rather than the triangular info icon seen in Figma)
  - but we may want to investigate in the future adding custom icons for some new designs (see Spoiler Alert in attached figma above)
- Make max width at 640px but the alert width adjusts based on the text content

@kenieh Let me know if I missed anything from our discussion! :)
